recipes/pysbol/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "pysbol" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/brsynth/pySBOL/archive/{{ version }}.tar.gz
+  sha256: 65d79b956e1249370307bc2a1e8638f776bd10b0559a6ab507477a65bcedc74c
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install -vv .
+  binary_relocation: true
+  missing_dso_whitelist:
+    - $RPATH/_libsbol.so
+    - $RPATH/libxml2-8ed442f7.so.2.6.26
+    - $RPATH/libz-a147dcb0.so.1.2.3
+    - $RPATH/libxslt-6cbe83bf.so.1.1.17
+    - /lib64/libc.so.6
+    - /lib64/libm.so.6
+    - /lib64/libdl.so.2
+    - /lib64/libm.so.6
+    - 'libstdcxx-ng'
+    - 'libgcc-ng'
+    - lib/libstdc++
+    - lib/libgcc_s.so.1
+    - lib/libstdc++.so.6
```

Comment:
you should no need any of this stuff. We cannot repackage binaries. you need to build the code in the recipe

Reply:
@beckermr , 
Do you mean build scripts with **build.sh** ? https://docs.conda.io/projects/conda-build/en/latest/resources/build-scripts.html
Do you have an example of a recipe or procedure to handle these binaries please?
Thanks a lot.

Reply:
I don't have an example handy.